packages/jaeger-ui/src/components/common/UiFindInput.tsx

```diff
@@ -115,4 +115,4 @@ export function extractUiFindFromState(state: ReduxState): TExtractUiFindFromSta
   return { uiFind };
 }
 
-export default withRouter(connect(extractUiFindFromState)(UnconnectedUiFindInput));
+export default withRouter<TOwnProps>(connect(extractUiFindFromState)(UnconnectedUiFindInput) as any);
```

Comment:
[Workaround ](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18999#:~:text=export%20default%20withRouter(connect(mapStateToProps%2C%20mapDispatchToProps)(PageContent)%20as%20any))to make it work with react-router v4. Since it will be upgraded to v6 in the future, this whole thing will be changed